src/main/webapp/javascript/dashboard.js

```diff
@@ -1,10 +1,10 @@
-let username = sessionStorage.getItem("username");
+let userId = sessionStorage.getItem("userId");
 
-if(username !== ""){
-    fetchTrips(username);
+if(userId !== ""){
+    fetchTrips(userId);
```

Comment:
Could we add an else statement that shows a pop up telling the user to sign in or log in and maybe include links.